src/test/postman/Scan_Upon_Demand_API.postman_collection.json

```diff
@@ -0,0 +1,93 @@
+{
+	"info": {
+		"_postman_id": "5773d958-5d6d-4680-aaf0-85074e8bff53",
+		"name": "Scan Upon Demand API",
+		"description": "Postman requests for the Scan Upon Demand API.",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+        {
+            "name": "Scan Upon Demand API",
+            "item": [
+                {
+                    "name": "Create Scan Upon Demand Item",
+                    "request": {
+                     "method": "POST",
+                     "header": [
+                            {
+                                "key": "Content-Type",
+                                "name": "Content-Type",
+                                "value": "application/json",
+                                "type": "text"
+                            }
+                        ],
+                     "body": {
+                        "mode": "raw",
+                        "raw": "{\n    \"company_number\": \"00006400\",\n    \"customer_reference\": \"Certificate ordered by OJ\",\n    \"item_options\": {\n        \"collection_location\": \"cardiff\",\n        \"contact_number\": \"02920123456\",\n        \"delivery_method\": \"postal\",\n        \"delivery_timescale\": \"same-day\",\n        \"filing_history_documents\": [\n            {\n             \"filing_history_id\": \"MDAxMTEyNzExOGFkaXF6a2N4\"\n            },\n            {\n             \"filing_history_id\": \"MzAwOTM2MDg5OWFkaXF6a2N4\"\n            },\n            {\n             \"filing_history_id\": \"MDE2OTkyOTEwMmFkaXF6a2N4\"\n            },\n            {\n             \"filing_history_id\": \"MDAyNzI3NTQ4OWFkaXF6a2N4\"\n            }\n        ],\n        \"forename\": \"bob\",\n        \"surname\": \"jones\"\n    },\n    \"quantity\": 1\n}",
+                        "options": {
```

Comment:
Good to see a Postman test in there too! :+1
We probably should align the request body with that in the spec?: https://developer-specs.cidev.aws.chdev.org/order-company-products/reference/scans-upon-demand-scud/create.

Reply:
Sure, yes on it